code/modules/admin/verbs/randomverbs.dm

```diff
@@ -781,7 +786,15 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(shuttle_master.emergency.mode >= SHUTTLE_DOCKED)
 		return
 
-	shuttle_master.emergency.cancel()
+	if(shuttle_master.emergency.canRecall == FALSE)
+		if(alert("Shuttle is currently set to be nonrecallable. Normally this happens due to round ending events. Respect Recall Status?", "Override Recall Status?", "Yes", "No") == "Yes")

```

Comment:
I think this would be better as, allow the admin to recall it without setting canRecall permanently, but warn them that they may break stuff if they do that.